scripts/update_android_ios_dependencies.py

```diff
@@ -71,7 +79,7 @@
 
 
 def get_files_from_directory(dirpath, file_extension, file_name=None,
-                             absolute_paths=True):
+                             absolute_paths=True, ignore_directories=None):
```

Comment:
What about `ignore_directories=[]` ?

Reply:
Ah... Using mutable values as defaults for arguments is a known python gotcha and can lead to unexpected behavior.
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

This pattern (default to None and set it to empty list in function) is used to avoid this.

Reply:
Interesting😔